skxray/core/roi.py

```diff
@@ -403,11 +403,10 @@ def mean_intensity(images, labeled_array, index=None):
     return mean_intensity, index
 
 
-def circular_average(image, calibrated_center, threshold=0, nx=100,
-                     pixel_size=None):
+def circular_average(image, calibrated_center, threshold=0, nx=None,
+                     pixel_size=(1, 1),  min_x=None, max_x=None, bin_width=1.):
```

Comment:
:-1: on bin width.  It leads to ambiguity if min_x, max_x and bin_width are all specified.